src/OpenLoco/Ui.h

```diff
@@ -106,30 +106,50 @@ namespace OpenLoco::Ui
     {
         enum class InteractionItem : uint8_t
         {
-            t_0 = 0,
-            t_1 = 1,
-            t_2 = 2,
-            thing = 3,
+            noInteraction = 0,
+            surface = 1,
+            industryTree = 2,
+            entity = 3,
             track = 4,
             trackExtra = 5,
             signal = 6,
             trackStation = 7,
             roadStation = 8,
             airport = 9,
             dock = 10,
-            t_11 = 11,
+            water = 11,
             tree = 12,
             wall = 13,
-            town = 14,
-            station = 15,
+            townLabel = 14,
+            stationLabel = 15,
             road = 16,
             roadExtra = 17,
-            t_18 = 18, // bridge?
+            bridge = 18,
             building = 19,
             industry = 20,
             headquarterBuilding = 21,
         };
 
+        namespace InteractionItemFlags // Bridge missing?
+        {
+            constexpr uint32_t surface = 1 << 0;
+            constexpr uint32_t entity = 1 << 1;
+            constexpr uint32_t track = 1 << 2;
+            constexpr uint32_t water = 1 << 3;
+            constexpr uint32_t tree = 1 << 4;
+            constexpr uint32_t road_and_tram = 1 << 5;
```

Comment:
stick to `camelCase` (yeah I know we aren't terribly consistent but that is what we are moving everything to.